cmds/record.c

```diff
@@ -19,6 +19,7 @@
 #include <sys/wait.h>
 #include <unistd.h>
 
+#include "libtraceevent/event-utils.h"
```

Comment:
I don't want to include libtraceevent headers just because of this.  Maybe we can implement `strim()` instead.

Reply:
will do this tonight